core/src/brute/bosses.ts

```diff
@@ -59,6 +66,30 @@ const bosses: Boss[] = [
     disarm: panther.disarm,
     damage: panther.damage,
     reach: 3,
+    count: 1,
+    reward: 1,
+    odds: 5,
+  },
+  {
+    name: BossName.Cerberus,
+    base: PetName.dog1,
+    scale: 2.15,
+    initiative: 1.5,
+    strength: dog1.strength * 7.5,
+    agility: dog1.agility,
+    speed: dog1.speed,
+    hp: 10000,
+    counter: dog1.counter,
+    combo: 0,
+    block: dog1.block,
+    evasion: -0.2,
+    accuracy: 0.75,
+    disarm: dog1.disarm,
+    damage: dog1.damage,
+    reach: 1,
+    count: 3,
+    reward: 0.2,
+    odds: 1,
```

Comment:
This should be set to 0 as it's only supposed to be the first boss, and not rolled afterwards